aiida/transports/plugins/ssh.py

```diff
@@ -437,14 +437,26 @@ def open(self):
             )
             raise
 
-        # Open also a SFTPClient
-        self._sftp = self._client.open_sftp()
-        # Set the current directory to a explicit path, and not to None
-        self._sftp.chdir(self._sftp.normalize('.'))
+        # Open also a File transport client. SFTP by default, pure SSH in ssh_only
+        self.open_file_transport()
+
+        return self
+
+    def open_file_transport(self):
+        """
+        Open the SFTP channel, and handle error by directing customer to try another transport
+        """
+        from aiida.common.exceptions import InvalidOperation
+
+        try:
+            self._sftp = self._client.open_sftp()
+        except Exception:
```

Comment:
Can we catch a more specific exception here? Is it predictable what would be raised? What did it raise in your case?

Reply:
in my case it was   paramiko.ssh_exception.SSHException: Channel closed. I'll catch only this, as it's apparently the one raised in this case. Done in 13fa71a